fgpyo/sam/__init__.py

```diff
@@ -547,6 +547,40 @@ def length_on_target(self) -> int:
         """Returns the length of the alignment on the target sequence."""
         return sum([elem.length_on_target for elem in self.elements])
 
+    def get_alignment_offsets(self, reverse: bool = False) -> Tuple[int, int]:
+        """
+                Get the starting and ending offset for the alignment based on the CIGAR string.
+
+                Args:
+                    reverse: Whether to count from the end of the read sequence.
+                        Default is False, which counts from the beginning of the read sequence.
+
+                Returns: Tuple[int, int] The start and end of the _aligned part_ of the read.
+                    0-based, open-ended offsets (to the read sequence.)
+                    If 'reverse' is True, the offsets count from the end of the read sequence.
+
+        # TODO: FIGURE OUT WHY the following three lines causes mkdocs to fail
```

Comment:
**FIXME** please fix and remove the TODO before merging

Reply:
I put it like this because I was unable to resolve...when I uncomment mkdocs barfs....

Reply:
```
  File "/Users/yossifarjoun/micromamba/envs/fgpyo/lib/python3.12/site-packages/griffe/docstrings/google.py", line 474, in _read_returns_section
    annotation = annotation.slice.elements[index]
                 ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
IndexError: list index out of range
```